CHANGELOG.rst

```diff
@@ -14,6 +14,9 @@ Fixed
 ^^^^^
 - Mixin does not work. (#1133)
 - `using_db` wrong position in model shortcut methods. (#1150)
+- Added `schema` attribute to Model's Meta to specify exact schema to use with the model.
```

Comment:
You should format like before, just add `Added` section

Reply:
ok, fixed